src/commands/initProjectForVSCode/detectProjectLanguage.ts

```diff
@@ -24,7 +24,10 @@ export async function detectProjectLanguage(context: IActionContext, projectPath
             if (await isTypeScriptProject(projectPath)) {
                 detectedLangs.push(ProjectLanguage.TypeScript);
             } else {
-                detectedLangs.push(ProjectLanguage.JavaScript);
+                // don't add JavaScript if TypeScript is already detected
+                if (!detectedLangs.includes(ProjectLanguage.TypeScript)) {
```

Comment:
`isTypeScriptProject` currently checks for `typescript` listed as a dev dependency. Isn't that required for TypeScript projects? At the end of the day, it seems fine if we prompt for JS vs TS for weird projects - and I think the unit tests might have weird projects

Reply:
If there's a `*/index.ts` file, but `typescript` is not a dev dependency, then we will detect JS and TS. Whereas before we only detected TS.

Although the test case is weird, if we find a `*/index.ts` file, I think it's totally safe to assume the project language isn't JS.

Reply:
No strong preference from me 🤷‍♂️